documentation.tex

```diff
@@ -0,0 +1,281 @@
+\documentclass[   % inherits article
```

Comment:
Please remove the associated PDF.

Reply:
Not sure why, but the README has the same content, so why not either.